### PR TITLE
[10.0][MIG] mrp_bom_structure_xlsx

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

File: mrp_bom_structure_xlsx/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': "Export BOM Structure to Excel .XLSX",
+    'version': '10.0.1.0.0',
+    'category': 'Manufacturing',
```
Comment (LoisRForgeFlow) [inline mrp_bom_structure_xlsx/__manifest__.py:9]: Add a little summary please

**Review by LoisRForgeFlow — APPROVED** [review]
Funcionally tested :+1:

File: mrp_bom_structure_xlsx/__manifest__.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply mrp_bom_structure_xlsx/__manifest__.py:9]: Done

Comment (lreficent) [inline mrp_bom_structure_xlsx/__manifest__.py]: :1st_place_medal:

Comment (LoisRForgeFlow) [reply mrp_bom_structure_xlsx/__manifest__.py:9]: quite brief but here is your :1st_place_medal: !

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional tests

**Comment by aheficent** [comment]
For everybody information:

Export BOM Structure to Excel .XLSX
=============================
This module extends the functionality of the MRP capabilities of Odoo, and allow you to export the BOM structure to MS Excel .XLSX format.

**Pull request merged**